base/src/main/java/com/smartdevicelink/proxy/rpc/GetVehicleDataResponse.java

```diff
@@ -1,620 +1,620 @@
-/*
- * Copyright (c) 2017 - 2019, SmartDeviceLink Consortium, Inc.
```

Comment:
@kostyaBoss It looks like the entire file `GetVehicleDataResponse.java` is being regenerated by the code generator tool.  Can you just merge in the changes for this PR (data related to `seatOccupancy`)?